src/openproduct/producttypen/urls.py

```diff
@@ -68,32 +76,139 @@
     ],
     "TAGS": [
-        {"name": "themas", "description": "Opvragen en bewerken van THEMA'S."},
         {
-            "name": "producttypen",
-            "description": "Opvragen en bewerken van PRODUCTTYPEN.",
+            "name": "themas",
+            "description": """
+## Opvragen en bewerken van THEMA'S.
+
+### Opmerkingen
+- Een thema kan onderdeel zijn van een ander thema, via `hoofd_thema` kan het hoofd thema worden gedefinieerd.
+- Een thema kan niet het hoofd thema van zichzelf zijn.
+- Een thema moet gepubliceerd zijn voordat zijn sub thema's kunnen worden gepubliceerd.
+- Een thema kan niet ongepubliceerd worden als het gepubliceerde sub thema's heeft.
+- Een thema niet worden verwijderd als het sub thema's heeft of als er door de verwijdering producttypen zonder thema's komen te zitten.
+""",
         },
-        {"name": "prijzen", "description": "Opvragen en bewerken van PRIJZEN."},
-        {"name": "links", "description": "Opvragen en bewerken van LINKS."},
-        {"name": "locaties", "description": "Opvragen en bewerken van LOCATIES."},
         {
-            "name": "organisaties",
-            "description": "Opvragen en bewerken van ORGANISATIES.",
+            "name": "producttypen",
+            "description": """
+## Opvragen en bewerken van PRODUCTTYPEN.
+
+### Opmerkingen
+- Bij het aanmaken en wijzigen van een producttype kan bij `thema_uuids`, `locatie_uuids`, `organisatie_uuids` & `contact__uuids` een lijst met uuids worden meegegeven.
+    - `thema_uuids` moet minimaal 1 uuid bevatten.
+    - Bij een PUT request word de bestaande lijst overschreven met de nieuwe lijst.
+    - Bij een PATCH request wordt de lijst alleen overschreven als het veld wordt meegegeven.
+- Bij het veld `uniforme_product_naam` gaat het om de naam uit de Uniforme productnamenlijst.
+- Bij de velden `verbruiksobject_schema_naam` en `dataobject_schema_naam` gaat het om de naam van een JSONSCHEMA.
+- De velden `externe_codes`, `parameters`, `zaaktypen`, `verzoektypen` en `processen` worden samen met het producttype genest aangemaakt of gewijzigd.
+    - Deze velden zijn een lijst van objecten.
+    - Bij een PUT request word de bestaande lijst overschreven met de nieuwe lijst.
+    - Bij een PATCH request wordt de lijst alleen overschreven als het veld wordt meegegeven.
+- Om de velden `zaaktypen`, `verzoektypen` en `processen` te gebruiken moet er in de ExterneVerwijzingConfig in de beheer interface de url voor elk object worden gedefinieerd.
+Tijdens het aanmaken/wijzigen wordt een uuid meegegeven. In de response zal deze uuid worden gecombineerd met de url uit de ExterneVerwijzingConfig.
+
+#### vertalingen
+- De velden `naam` en `samenvatting` zijn meertalig, waarbij Nederlands verplicht is en Engels optioneel is.
+- Via de reguliere create & update methodes kunnen alleen de Nederlands teksten worden aangemaakt/gewijzigd.
+- Via producttypen/<uuid>/vertaling/<taal> kan de engelse variant worden gewijzigd.
```

Comment:
This doesn't seem to render properly, because it's interpreted as an html tag now I think

![afbeelding](https://github.com/user-attachments/assets/df6f3316-0f13-4031-8471-31605077403b)


```suggestion
- Via `producttypen/<uuid>/vertaling/<taal>` kan de engelse variant worden gewijzigd.
```